encoding/protobuf/cue/Makefile

```diff
@@ -1,7 +1,20 @@
-all: build
+all: install-go-tools build
+
+ROOTDIR := $(shell pwd)
+OUTPUTDIR = $(ROOTDIR)/_output
+DEPSGOBIN = $(OUTPUTDIR)/.bin
+
+.PHONY: mod-download
+mod-download:
+	go mod download
+
+.PHONY: install-go-tools
+install-go-tools: mod-download
+	mkdir -p $(DEPSGOBIN)
+	GOBIN=$(DEPSGOBIN) go install github.com/golang/protobuf/protoc-gen-go
 
 build:
-	mkdir -p ./_output
-	protoc -I=. -I=./imports/ --go_out=./_output cue.proto
-	cp ./_output/cuelang.org/go/encoding/protobuf/cue/cue.pb.go .
-	rm -rf ./_output
\ No newline at end of file
+	mkdir -p $(OUTPUTDIR)
+	PATH=$(DEPSGOBIN):$$PATH protoc -I=. -I=./imports/ --go_out=$(OUTPUTDIR) cue.proto
```

Comment:
`-I=. -I=./imports`: do we want to use $ROOTDIR instead of `.`?

Reply:
Good call yeah I can update